src/components/form-attachment/list.vue

```diff
@@ -420,6 +439,10 @@ export default {
   // Extend to the bottom of the page (or slightly beyond it) so that the drop
   // zone includes the entire page below the PageHead.
   min-height: calc(100vh - 146px);
+
+  .panel-dialog {
+    margin-bottom: 20px;
```

Comment:
Right now, the `.panel-dialog` is closer to the table than to the text above it, when my instinct is that it should be the other way around. What do you think about something like:

```scss
margin-top: -5px;
margin-bottom: 25px;
```